core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1529,6 +1528,62 @@ table.tl_listing {
 	margin-left: 1px;
 }
 
+.operations > ul {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	gap: 3px;
+	flex-wrap: wrap;
+	list-style: none;
```

Comment:
```suggestion
	list-style: none;
	margin: 0;
```

We would want to always disable the browser styles for this offset to not happen :)
![image](https://github.com/user-attachments/assets/26a04e38-9100-4a9e-8232-452fb92472a4)